Files/MultilingualResources/Files.es-ES.xlf

```diff
@@ -2729,171 +2724,171 @@
         </trans-unit>
         <trans-unit id="FileBrowserThumbnailIconColumnGrid.AutomationProperties.Name" translate="yes" xml:space="preserve">
           <source>File icon</source>
-          <target state="new">File icon</target>
+          <target state="translated">Ícono</target>
```

Comment:
It should be "Icono" as "Ícono" is the way written in Spanish in America ([(https://dle.rae.es/icono)])